lua/tardis/interiors/default.lua

```diff
@@ -1,124 +1,124 @@
--- Default
+-- Default (used to be Legacy)
```

Comment:
This wording is not necessary

```suggestion
-- Default
```